webSocketFunctions.js

```diff
@@ -35,6 +42,7 @@ const assignRoles = (game) => {
     player.role = role;
     player.title = roleInfo.title;
     player.ability = roleInfo.ability;
+    player.url=roleInfo.url;
```

Comment:
Nitpick: The spacing of this line doesn't follow the above lines. Changing this to match the others keeps the file more consistent and keeps the code base 'cleaner'.

Reply:
Done